index.html

```diff
@@ -3350,9 +3308,8 @@ <h3>JSON-LD</h3>
 
         <p>
 [[!JSON-LD]] is useful when extending the data model described in this
-specification. Instances of the data model are encoded in [[!JSON-LD]] in the
-same way they are encoded in JSON (Section <a href="#json"></a>), with the
-addition of the <code>@context</code> <a>property</a>. The
+specification. Instances of the data model are encoded in [[!JSON-LD]]
+and include the <code>@context</code> <a>property</a>. The
```

Comment:
Do we want to comment on compact JSON-LD here?

Reply:
Probably.

Reply:
Fixed in 3ef0175a4931bafc30dd73f893889e6f6ce25c8b.